api/v1.inc

```diff
@@ -31,10 +31,13 @@ $router->add_route("GET", "v1/projects/imagesources", "api_v1_projects_imagesour
 $router->add_route("GET", "v1/projects/:projectid/pages", "api_v1_project_pages");
 $router->add_route("GET", "v1/projects/:projectid/pagedetails", "api_v1_project_pagedetails");
 $router->add_route("GET", "v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid", "api_v1_project_page_pageround");
-$router->add_route("PUT", "v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid", "api_v1_project_page_pageround");
 $router->add_route("GET", "v1/projects/:projectid/codepoints", "api_v1_project_codepoints");
 
 $router->add_route("POST", "v1/projects/:projectid/pages/checkout", "api_v1_project_checkout");
+$router->add_route("GET", "v1/projects/:projectid/pages/:pagename/pageoptions/:pageoption", "api_v1_project_page_option");
+$router->add_route("PATCH", "v1/projects/:projectid/pages/:pagename", "api_v1_project_page");
+$router->add_route("PUT", "v1/projects/:projectid/pages/:pagename", "api_v1_project_page");
+$router->add_route("DELETE", "v1/projects/:projectid/pages/:pagename", "api_v1_project_page");
```

Comment:
As a user, I would expect this to delete a page but it does't, it only change the state of the page. We should not use the DELETE verb for this.

Reply:
My comment below was later than yours. But if we specify the resource in the url as  the page state then it will be clearer what we are deleting.

Reply:
I'm also uncomfortable using the DELETE verb. Isn't there something that describes it better that can be used?

Reply:
There are only a small number of verbs available. I think if we use a url perhaps including something which shows clearly this is a page checked out to a user it would be in the spirit of restful to delete it. Or we can move this function into the PUT case, i.e. put the state to available. We would be returning a page state which would be unused but that is not a problem. I think it is not required to have the same input to all cases of the same PUT url since these can be optional. (The other cases of PUT are saves with page text input).